examples/2d/2d_shapes.rs

```diff
@@ -18,36 +18,47 @@ fn setup(
 
     // Circle
     commands.spawn(MaterialMesh2dBundle {
-        mesh: meshes.add(shape::Circle::new(50.)).into(),
-        material: materials.add(Color::PURPLE),
-        transform: Transform::from_translation(Vec3::new(-150., 0., 0.)),
+        mesh: meshes.add(primitives::Circle { radius: 50.0 }).into(),
+        material: materials.add(Color::VIOLET),
+        transform: Transform::from_translation(Vec3::new(-225.0, 0.0, 0.0)),
         ..default()
     });
 
-    // Rectangle
-    commands.spawn(SpriteBundle {
-        sprite: Sprite {
-            color: Color::rgb(0.25, 0.25, 0.75),
-            custom_size: Some(Vec2::new(50.0, 100.0)),
-            ..default()
-        },
-        transform: Transform::from_translation(Vec3::new(-50., 0., 0.)),
+    // Ellipse
+    commands.spawn(MaterialMesh2dBundle {
+        mesh: meshes.add(primitives::Ellipse::new(50.0, 100.0)).into(),
+        material: materials.add(Color::TURQUOISE),
+        transform: Transform::from_translation(Vec3::new(-100.0, 0.0, 0.0)),
         ..default()
     });
 
-    // Quad
+    // Rectangle
     commands.spawn(MaterialMesh2dBundle {
-        mesh: meshes.add(shape::Quad::new(Vec2::new(50., 100.))).into(),
+        mesh: meshes.add(primitives::Rectangle::new(50.0, 100.0)).into(),
         material: materials.add(Color::LIME_GREEN),
-        transform: Transform::from_translation(Vec3::new(50., 0., 0.)),
+        transform: Transform::from_translation(Vec3::new(0.0, 0.0, 0.0)),
         ..default()
     });
 
     // Hexagon
     commands.spawn(MaterialMesh2dBundle {
-        mesh: meshes.add(shape::RegularPolygon::new(50., 6)).into(),
-        material: materials.add(Color::TURQUOISE),
-        transform: Transform::from_translation(Vec3::new(150., 0., 0.)),
+        mesh: meshes.add(primitives::RegularPolygon::new(50.0, 6)).into(),
+        material: materials.add(Color::YELLOW),
+        transform: Transform::from_translation(Vec3::new(125.0, 0.0, 0.0)),
+        ..default()
+    });
+
+    // Triangle
+    commands.spawn(MaterialMesh2dBundle {
+        mesh: meshes
+            .add(primitives::Triangle2d::new(
+                Vec2::Y * 50.0,
+                Vec2::new(-50.0, -50.0),
+                Vec2::new(50.0, -50.0),
+            ))
+            .into(),
```

Comment:
I swear we made these `.into` calls no longer required.

Reply:
This is `meshes.add(mesh).into()`, not `meshes.add(mesh.into())` `MaterialMesh2dBundle` requires a `Mesh2dHandle`, so we need to convert `Handle<Mesh>` to that.

Prior to the `.into()` PR this would've been `meshes.add(mesh.into()).into()`